### DISCUSSION TITLE
Is there a guide to understand the files when modifying the source code in a local server?

### DISCUSSION THREAD

**Discussion by animanoir** [mozilla/hubs#2786] [General Questions & Discussion] [answered]
I want to make some custom changes in the app, but it's enormous! Where can I find the index and all the simple stuff, hehe.

**Comment by brianpeiris** [comment, marked as answer]
Hi @animanoir, I'm afraid there isn't an overview of the source code at the moment. In the future we intend to have specific extension points to make it easier for other developers to customize Hubs. In the meantime, we can help you navigate through the code a bit. It would help if you asked specific questions about what customizations you're trying to make so that we can guide you to the right parts of the code.

**Reply by ocularvr** [reply]
Brian we are interested in making this document for the community. We are using a AWS implementation and would really like to help. There are lots of documents to be made - AWS setup (current one has errors like email ports), Github, Rebuilding source, Deploying stack. This would be what we would like to document. For example if you tell me the development environment, we will download source and write all the details in a document. Similarly, if you explain best practices for content delivery, we will set it up, then write it up. Let me know if this is possible please.
